bittensor/config.py

```diff
@@ -128,7 +129,7 @@ def __str__(self):
             self.chain_endpoint, self.neuron_key, self.axon_port, self.metagraph_port,
             self.metagraph_size, self.bootstrap, self.remote_ip, self.datapath, self.logdir)
 
-    def from_hparams(hparams):
+    def from_hparams(self, hparams):
```

Comment:
This shouldn't be a non-static function because we call it on bittensor.Config.from_hparams(hparams) to return a new config object. Pretty sure this will cause an error in the mnist tests?

Reply:
Good point. I changed it because i didn't see an `@staticmethod` in there. Added it in, and removed the `self`